xknx/knxip/cemi_frame.py

```
@@ -79,11 +83,21 @@ def telegram(self, telegram: Telegram) -> None:
             CEMIFlags.FRAME_TYPE_STANDARD
             | CEMIFlags.DO_NOT_REPEAT
             | CEMIFlags.BROADCAST
-            | CEMIFlags.PRIORITY_LOW
             | CEMIFlags.NO_ACK_REQUESTED
             | CEMIFlags.CONFIRM_NO_ERROR
             | CEMIFlags.HOP_COUNT_1ST
         )
+        if telegram.priority == Priority.SYSTEM:
+            # do nothing
+            pass
+        elif telegram.priority == Priority.URGENT:
+            self.flags |= CEMIFlags.PRIORITY_URGENT
+        elif telegram.priority == Priority.NORMAL:
+            self.flags |= CEMIFlags.PRIORITY_NORMAL
+        elif telegram.priority == Priority.LOW:
+            self.flags |= CEMIFlags.PRIORITY_LOW
+        else:
+            raise RuntimeError("Unknown telegram priority: " + str(telegram.priority))
```

Please raise a domain exception like `XKNXException`.